ArduCopter/mode_auto.cpp

```diff
@@ -185,7 +181,7 @@ void ModeAuto::takeoff_start(const Location& dest_loc)
     }
 
     // set waypoint controller target
-    if (!wp_nav->set_wp_destination(dest)) {
+    if (!wp_nav->set_wp_destination_loc(dest)) {
```

Comment:
I think @lthall found the polymorphism confusing.  I'm reluctant to rename it back (or change it to "set_destination") because I think it's more important that Leonard finds it easy to work with.  I agree removing the "_wp_" part of "set_wp_destination_loc" is an improvement but I think I will leave it as-is for now because going back and renaming it again is just too painful.